Nodify/Connections/BaseConnection.cs

```diff
@@ -205,7 +241,23 @@ protected override Geometry DefiningGeometry
 
                     if (ArrowSize.Width != 0d && ArrowSize.Height != 0d)
                     {
-                        DrawArrowGeometry(context, arrowSource, arrowTarget);
+                        switch(ArrowEnds)
+                        {
+                            case ArrowHeadEnds.Start: 
+                                DrawArrowGeometry(context, arrowTarget, arrowSource, ConnectionDirection.Backward);
+                                break; 
+                            case ArrowHeadEnds.End:
+                                DrawArrowGeometry(context, arrowSource, arrowTarget);
```

Comment:
I know there's a default value for the direction argument, but I think that in this case it would be better to specify it because it's important to control the value from here im case the default value changes in the future.